### ISSUE TITLE
Can ipadapter handle transparent backgrounds?

### ISSUE THREAD

**Issue by Charuru** [tencent-ailab/IP-Adapter#106]
Question here: https://www.reddit.com/r/comfyui/comments/17ailay/ipadapter_working_with_alpha_or_how_to_use/

Trying to make the end result not have to deal with the black bg influencing the image.

https://i.imgur.com/GG3UO7w.png

**Comment by xiaohu2015** [comment]
@Charuru now IP-Adapter don't support that (As far as I know, currently SD only supports RGB images). I think you can convert the bg to white bg instead of black bg?

**Comment by Charuru** [comment]
@xiaohu2015 Thanks, white background has a similar problem with the image being influenced. It sounds like it's unfixable right now?

**Comment by xiaohu2015** [comment]
@Charuru yes, even for text prompt, the background is not completely controllable
